gcp_variant_transforms/transforms/merge_headers.py

```diff
@@ -42,5 +190,13 @@ def extract_output(self, merged_headers):
 class MergeHeaders(beam.PTransform):
   """A PTransform to merge VCF file headers."""
 
+  def __init__(self, split_alternate_allele_info_fields=True):
+    super(MergeHeaders, self).__init__()
+    self._split_alternate_allele_info_fields = (
```

Comment:
Please remove from self as it's no longer needed.

Reply:
Done